packages/calendar-time-utils/src/formats.ts

```diff
@@ -0,0 +1,69 @@
+// Reference: https://momentjs.com/docs/#/displaying/format/
+export enum FormatToken {
```

Comment:
We usually don't use enums, as there are issues with Babel (as far as I remember).

You can use instead the `as const` syntax.

```ts
const formatToken = {
  D: 'D',
  // ...
} as const
```

Reply:
Updated in this commit: 57e106548f5d93026e3a5fb8f5d2b42b66f7653c